InvenTree/plugin/test_api.py

```diff
@@ -193,8 +193,3 @@ def test_check_plugin(self):
         with self.assertRaises(NotFound) as exc:
             check_plugin(plugin_slug=None, plugin_pk='123')
         self.assertEqual(str(exc.exception.detail), "Plugin '123' not installed")
-
-        # Not active
-        with self.assertRaises(NotFound) as exc:
```

Comment:
@SchrodingersGat was this test not working or why was it removed? I think it would make sense to check for this.

Reply:
As this plugin is now always "active" (because it is a builtin plugin) the test was no longer working